docs/concepts/storage/volumes.md

```diff
@@ -170,6 +171,36 @@ writers simultaneously.
 
 See the [CephFS example](https://github.com/kubernetes/examples/tree/{{page.githubbranch}}/staging/volumes/cephfs/) for more details.
 
+### csi
+
+CSI stands for [Container Storage Interface](https://github.com/container-storage-interface/spec/blob/master/spec.md),
+a specification attempting to establish an industry standard interface that
+Container Orchestration Systems (COs) can use to expose arbitrary storage systems
+to their container workloads.
```

Comment:
Could you add a "Also See" or a link to https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md ?